src/main/scala/uk/gov/homeoffice/cirium/CiriumFlightStatusApp.scala

```diff
@@ -2,173 +2,63 @@ package uk.gov.homeoffice.cirium
 
 import akka.actor.{ ActorRef, ActorSystem, Scheduler }
 import akka.http.scaladsl.Http
-import akka.http.scaladsl.model.{ HttpResponse, StatusCodes }
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
-import akka.http.scaladsl.server.directives.MethodDirectives.get
-import akka.http.scaladsl.server.directives.RouteDirectives.complete
-import akka.pattern.{ AskableActorRef, ask }
 import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.Sink
-import akka.util.Timeout
 import com.typesafe.scalalogging.Logger
-import uk.gov.homeoffice.cirium.actors.CiriumFlightStatusRouterActor.GetReadiness
-import uk.gov.homeoffice.cirium.actors.CiriumPortStatusActor.{ GetStatuses, GetTrackableStatuses }
 import uk.gov.homeoffice.cirium.actors.{ CiriumFlightStatusRouterActor, CiriumPortStatusActor }
-import uk.gov.homeoffice.cirium.services.entities.{ CiriumFlightStatus, CiriumTrackableStatus }
-import uk.gov.homeoffice.cirium.services.feed.{ Cirium, Retry }
-import uk.gov.homeoffice.cirium.services.health.{ AppHealthCheck, CiriumAppHealthSummaryConstructor }
+import uk.gov.homeoffice.cirium.services.api.{ FlightStatusRoutes, FlightScheduledRoutes, StatusRoutes }
+import uk.gov.homeoffice.cirium.services.feed.Cirium
 
-import scala.concurrent.duration.{ Duration, _ }
+import scala.concurrent.duration.Duration
 import scala.concurrent.{ Await, ExecutionContext, Future }
 import scala.language.postfixOps
 import scala.util.{ Failure, Success }
 
-object CiriumFlightStatusApp extends App {
-  val log = Logger(getClass)
+trait Environment {
```

Comment:
I think extracting these out is a good idea, but might be better to put them on a global object so that they can be accessed without needing to implement a trait. Having them all in one place is also handy.